docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+
+services:
+  postgres:
+    environment:
+      - POSTGRES_USER=metaci
+      - POSTGRES_DB=metaci
+      - POSTGRES_PASSWORD=
+    image: postgres:10.6
+    ports: 
+    - "5432:5432/tcp"
+    restart: always
+  redis:
+    image: 'redis:5'
+    ports:
+      - '6379:6379'
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: |
+        /bin/bash -c 'while !</dev/tcp/postgres/5432; do /bin/sleep 1; done; \
+        /bin/sleep 1; /bin/sh /app/utility/start_server.sh;'
+    ports:
+      - "8000:8000"
+    depends_on:
+      - postgres
+      - redis
+    env_file:
+      - .env
+    environment:
+      - BUILD_ENV=development
```

Comment:
BUILD_ENV is an argument which should go in the `args` setting under `build`, not an environment variable used when the container is running. https://docs.docker.com/compose/compose-file/#args

Reply:
So I tried this but its not passed in for whatever reason, so keeping other way for now would love to fix.